axelor-quality/src/main/resources/views/QualityPictures.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<object-views xmlns="http://axelor.com/xml/ns/object-views" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+	xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_5.0.xsd">
+
+	<grid name="quality-pictures-grid" title="Pictures" model="com.axelor.apps.quality.db.QualityPictures">
+		<field name="name"/>
+		<field name="content"/>
+		<field name="createDate"/>
+	</grid>
+
+	<form name="quality-pictures-form" title="Pictures" model="com.axelor.apps.quality.db.QualityPictures" onNew="action-quality-picture-record-on-new">
+		<panel name="mainPanel">
+			<panel name="imagePanel" colSpan="6">
+				<field name="content" widget="Image" colSpan="12"/>
+			</panel>
+			<panel name="detailsPanel" colSpan="6">
+				<field name="name" colSpan="12"/>
+				<field name="createDate" colSpan="6"/>
+				<field name="qualityMeasuringPoint" hidden="true"/>
+			</panel>
+		</panel>
+	</form>
+
+	<action-record name="action-quality-picture-record-on-new" model="com.axelor.apps.quality.db.QualityPictures">
+		<field name="qualityMeasuringPoint" expr="eval: _parent" if="_parent?._model == 'com.axelor.apps.quality.db.QualityMeasuringPoint'"/>
```

Comment:
What is the need to set qualityMeasuringPoint ?

Reply:
Mistake, not needed.